config/routes.rb

```diff
@@ -40,4 +40,7 @@
   get "/users/new", to: "users#new"
   post "/users/new", to: "users#create"
   get "/users/:id", to: "users#show"
+
+  get '/admin', to: 'admins#dashboard'
```

Comment:
I think the routes here are a bit off, I dont think dashboard is supposed to be an action, I believe the route would look like this: 
`get '/merchant/dashboard', to: "dashboard#index"`